pdfminer/cmapdb.py

```diff
@@ -425,24 +457,20 @@ def do_keyword(self, pos: int, token: PSKeyword) -> None:
         if token is self.KEYWORD_BEGINNOTDEFRANGE:
             self.popall()
             return
+
         if token is self.KEYWORD_ENDNOTDEFRANGE:
             self.popall()
             return
 
         self.push((pos, token))
-        return
-
-
-def main(argv: List[str]) -> None:
```

Comment:
By removing this, we're making it so you can no longer run this file directly from the command line. Is this intentional? Should it be in the changelog?

Reply:
It was intentional. 

But I see there are many files that can be run directly. 

I guess it was initially made for testing purposed.

I'll create an issue to deprecate, remove and (potentially) replace this functionality. Restored it for now.